tests/v2/test_waku_rln_relay.nim

```diff
@@ -628,9 +632,79 @@ suite "Waku rln relay":
 
     # verify the proof (should not be verified)
     let verified = rln.proofVerify(data = messageBytes,
-                                 proof = proof)
+                                  proof = proof)
+    check:
+      verified.isOk()
+      verified.value() == false
+
+  test "invalidate messages with a valid, but stale root":
+    var rlnInstance = createRLNInstance()
     check:
-      verified == false
+      rlnInstance.isOk == true
```

Comment:
If this is a condition that if it fails, it will fail in many tests, then it is preferable to use the `require()` assertion method.

Reply:
Addressed in [5d48c46](https://github.com/status-im/nwaku/pull/1158/commits/5d48c46018f964874d9404006398d3505ebf3728)